modules/cnf-du-partitioning-management-workloads.adoc

```diff
@@ -0,0 +1,43 @@
+where you want to reserve most of the CPU resources for user workloads and configure OpenShift to run on a fixed
+number of CPUs within the host.
+
+Server resources deployed at the edge, such as cores and memory, are expensive and limited. Customers require nearly all cores for their application workloads. The resources consumed by infrastructure is a key portion of the customer's business case for the selection of a vRAN infrastructure.
+
+A hypothetical DU example is an unusually resource-intensive workload, typically requiring 20 dedicated cores, 24 GiB of RAM consumed as huge pages, multiple SR-IOV NICs carrying several Gbps of traffic each, and specialized accelerator devices. The node hosting this workload must run a realtime kernel, and be carefully tuned to ensure low-latency requirements can be met, and be configured to support features such as Precision Timing Protocol (PTP).
+
+This feature implements the necessary OCP features and cluster configuration to isolate all housekeeping and infrastructure pods to a partitioned set of CPU resources on a Single Node OpenShift (SNO) cluster with a DU profile applied. This falls broadly into two categories:
+
+* Isolates cluster management functions to the defined number of CPUs. All housekeeping tasks and infrastructure pods operate solely on that cpuset.
+
+* Tunes the cluster configuration (with the applied DU profile) so the actual CPU usage fits within the assigned cpuset.
+
+[NOTE]
+====
+This feature is only available on SNO in this release.
+====
+
+The minimum number of reserved CPUs required for the management partition for an SNO is four CPUs/HTs. Inclusion of operators or workloads beyond the set specified in “Management pods” requires additional CPU/HTs.
+
+Workload partitioning isolates the workloads away from the non-management workloads using the normal scheduling capabilities of kubernetes to manage the number of pods that can be placed onto those cores, and avoids mixing management and normal workloads.
+
+To fully leverage this feature, you need to deploy the Performance Addon Operator and apply the performance profile as part of day 2 configuration.
+
+The concept of management workloads is flexible and can encompass:
+
+* All OpenShift core components necessary to run the cluster.
+
+* Any add-on operators necessary to make up the platform as defined by the telco customer.
+* Operators or other components from third-party vendors that the customer deems as management rather than operational.
+** Adding management workloads may require additional CPU resources to be added to the partition.
+
+This feature introduces a new extended resource of `{workload-type}.workload.openshift.io/cores` for each CPU pool (workload-type) defined in the configuration file.  Kubelet advertises this new resources. When the feature is enabled it represents all of the CPU capacity of the host, not just the CPU pool.
```

Comment:
Consider partition rather than pool for consistency

mismatched "this" and "new resources"

Reply:
Fixed.